src/backend/distributed/metadata/node_metadata.c

```diff
@@ -1537,6 +1539,25 @@ citus_nodeport_for_nodeid(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * citus_coordinator_node_id returns the node id of the coordinator node
+ */
+Datum
+citus_coordinator_node_id(PG_FUNCTION_ARGS)
```

Comment:
maybe node_id -> nodeid to be consistent with column names and other functions (e.g. get_nodeid_for_groupid)